src/main/java/bio/terra/service/policy/exception/PolicyServiceAPIException.java

```diff
@@ -0,0 +1,47 @@
+package bio.terra.service.policy.exception;
+
+import bio.terra.buffer.client.ApiException;
+import bio.terra.common.exception.ErrorReportException;
+import java.util.Collections;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+
+/** Wrapper exception for non-200 responses from calls to Terra Policy Service. */
+public class PolicyServiceAPIException extends ErrorReportException {
```

Comment:
Style nit: [Google Java style guide](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case) advises camel casing of acronyms.  Under that style rule, this would be `PolicyServiceApiException`.